packages/std/src/results/submessages.rs

```diff
@@ -196,13 +201,20 @@ impl From<SubMsgResult> for Result<SubMsgResponse, String> {
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
 pub struct SubMsgResponse {
     pub events: Vec<Event>,
+    #[deprecated = "deprecated in the cosmos-sdk in favor of msg_responses"]
     pub data: Option<Binary>,
+    #[serde(default)]
+    pub msg_responses: Vec<MsgResponseValue>,
 }
 
-#[deprecated(note = "Renamed to SubMsgResponse")]
-pub type SubMsgExecutionResponse = SubMsgResponse;
+#[derive(Serialize, Deserialize, Clone, Debug, PartialEq, Eq, JsonSchema)]
+pub struct MsgResponseValue {
```

Comment:
Ah here we have what we discussed already. Mybe just drop the "Value" as it does not add anything as far as I can tell.